dds/DCPS/Service_Participant.cpp

```diff
@@ -578,6 +584,56 @@ Service_Participant::get_domain_participant_factory(int &argc,
   return DDS::DomainParticipantFactory::_duplicate(dp_factory_servant_.in());
 }
 
+
+
+void Service_Participant::parse_env()
+{
+#if defined (ACE_WIN32)
+  LPTCH env_strings = GetEnvironmentStrings();
+
+  // If the returned pointer is NULL, exit.
+  if (!env_strings) {
+    if (log_level >= LogLevel::Error) {
+      ACE_ERROR((LM_ERROR,
+                 "(%P|%t) ERROR: Service_Participant::parse_env: Could not get environment strings\n"));
+    }
+    return;
+  }
+
+  LPTSTR env_string = (LPTSTR) env_strings;
+
+  while (*env_string) {
+    parse_env(ACE_TEXT_ALWAYS_CHAR(env_string));
```

Comment:
@mitza-oci Does this make sense?

Reply:
Try adding a uses_wchar Windows build to CI

Reply:
For posterity...

build_and_test.yml is at the limit that GHA will allow so we will need to refactor it before we can add another build.

Converting an existing Windows build failed with build errors.  One of the root causes was wide character not being supported correctly for ACE's XML support.  So, we will need to be selective about what features to include in such a build.

I will record these points in the Wiki.